app/views/questions/consent/edit.html.erb

```diff
@@ -9,14 +9,9 @@
 
       <%= f.cfa_input_field(:primary_first_name, t("views.questions.consent.primary_first_name")) %>
       <%= f.cfa_input_field(:primary_last_name, t("views.questions.consent.primary_last_name")) %>
-      <%= f.cfa_input_field(
-              :primary_last_four_ssn,
-              t("views.questions.consent.primary_last_four_ssn"),
-              prefix: "XXX-XX-",
-              type: :tel,
-              classes: ["form-width--name field--last-four-ssn"],
-              options: { maxlength: 4 }
-          ) %>
+    <%= f.cfa_select(:primary_tin_type, t('general.tin_type'), tin_options_for_select(include_itin: true)) %>
+    <%= f.cfa_input_field(:primary_ssn, t("attributes.primary_ssn"), classes: ["form-width--long"], options: { maxlength: 11 }) %>
```

Comment:
is the maxlength 11 to include the 2 dashes?

Reply:
yep!